docs.md

```diff
@@ -850,7 +850,7 @@ The table of contents automatically generated from the headers should give learn
 This section will appear in any module that requires the learner to prepare in some way. For example:
 - the learner needs to download software (like git or bash)
```

Comment:
```suggestion

- the learner needs to download software (like git or bash)
```

There was just a missing newline here, preventing the list from rendering properly.